SygicMapsKit/Classes/SearchModule/SYMKSearchViewController.swift

```diff
@@ -65,6 +65,7 @@ public class SYMKSearchViewController: SYMKModuleViewController {
     // MARK: - Public methods
     
     override func sygicSDKInitialized() {
+        model.search = SYSearch()
```

Comment:
```suggestion
        model = SYMKSearchModel()
```

Reply:
no.. neviem preco som to napisal takto .. :D

Reply:
Jaj, uz viem preco som to spravil takto .. lebo metody `searchCoordinates(SYGeoCoorinates)` a `maxResults(UInt)` nastavaju modelu tieto atributy. A tie chces zvycajne nastavit pri inicializacii search modulu